src/Symfony/Component/String/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+7.2
+---
+
+ * Add `TruncateMode` enum to handle more truncate methods
```

Comment:
How we map the bools to the enum cases, should be added to the UPGRADE guide

Reply:
https://github.com/symfony/symfony/pull/57243/files#diff-cbf6afe492929e3f85df879b4ca89f8c1fc2131eb3a7830d65873612f2bf54a4R16-R22
Does that looks ok to you ?